infra/app/service/main.tf

```diff
@@ -35,6 +35,9 @@ locals {
 
   service_name = "${local.prefix}${module.app_config.app_name}-${var.environment_name}"
 
+  # Include project name in bucket name since buckets need to be globally unique across AWS
+  bucket_name = "${local.prefix}${module.project_config.project_name}-${module.app_config.app_name}-${var.environment_name}"
```

Comment:
@lorenyu @shawnvanderjagt I was thinking about this the other day and was wondering if this should include a [random ID](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/id) as part of its name. S3 bucket names have to be globally unique, so I think there's a small chance that this name could conflict with an existing bucket name. For instance, a team trying to create a `pfml-portal-dev` bucket, which is used by some other team already (Nava or elsewhere). 

I think I've seen the AWS CDK generate buckets using a similar convention of appending a random string.

Reply:
Good point. Added a ticket. Will think about the implementation a bit. https://github.com/navapbc/template-infra/issues/520